zerver/lib/notifications.py

```diff
@@ -101,11 +101,14 @@ def relative_to_full_url(content):
         content = re.sub(
             r"<img src=(\S+)/user_uploads/(\S+)>", "", content)
 
-        # URLs for emoji are of the form
-        # "static/generated/emoji/images/emoji/snowflake.png".
+        # Change relative URLs for emoji images to absolue URLs. Relative
+        # URLs for emoji images are of the form:
+        # "static/generated/emoji/images/emoji/unicode/<emoji-code>.png".
+        # Don't replace the URLs for the zulip emoji, we need to serve it
+        # from the old emoji farm.
         content = re.sub(
-            r"/static/generated/emoji/images/emoji/",
-            user_profile.realm.uri + r"/static/generated/emoji/images/emoji/",
+            r"/static/generated/emoji/images/emoji/unicode/(?!zulip\.png)",
+            user_profile.realm.uri + r"/static/generated/emoji/images-google-64/",
```

Comment:
I guess we'll want to change this to a more complicated substitution from the new emoji spans when we merge the other PR, right?

Reply:
Yep. :)

Reply:
Now as I thought more about it, I think I will be able to solve the alt code misalignment problem while making that substitution.